ChartboostMediationDemo/Demo_ObjC/Fullscreen/FullscreenAdController.m

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023-2024 Chartboost, Inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file.
+
+#import "FullscreenAdController.h"
+
+@interface FullscreenAdController ()
+
+@property (nonatomic, weak, nullable) id<ActivityDelegate> activityDelegate;
+@property (nonatomic, strong, nullable) id<ChartboostMediationFullscreenAd> fullscreenAd;
+
+@end
+
+@implementation FullscreenAdController
+
+- (nonnull instancetype)initWithPlacementName:(nonnull NSString *)placementName activityDelegate:(id<ActivityDelegate> _Nullable)activityDelegate {
+    self = [super self];
+    if (self) {
+        _placementName = placementName;
+        _activityDelegate = activityDelegate;
+    }
+    return self;
+}
+
+- (void)load {
+    [self loadWithKeywords:nil];
+}
+
+- (void)loadWithKeywords:(HeliumKeywords * _Nullable)keywords {
+    // Attempt to load the ad only if it has not already been created and requested to load.
+    if (self.fullscreenAd) {
+        NSLog(@"[Warning] fullscreen advertisement has already been loaded");
+        return;
+    }
+
+    // Notify the demo UI
+    [self.activityDelegate activityDidStart];
+
+    // loadFullscreenAd expects keywords to be `[String : String]` instead of `HeliumKeywords?`.
```

Comment:
```suggestion
    // Optional keywords that can be associated with the advertisement placement.
```

I'm thinking we just completely change this comment, since no part of the old comment really makes sense here.